.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,3 @@
-# Pull Request Type
```

Comment:
This isn't related, please do this separately.

Reply:
ok